fairlearn/datasets/_fetch_acs_income.py

```diff
@@ -7,8 +7,6 @@
 import pandas as pd
 from sklearn.datasets import fetch_openml
 
-import fairlearn.utils._compatibility as compat
```

Comment:
should we then remove `_PARSER_KWARGS`?

Reply:
Probably.... comments there suggest it was a v1.2 thing, if you're happing bumping up that far.

Reply:
@adrinjalali I've bumped to 1.2 below.... if you'd prefer to stick to 1.1, I'll need to change how compatibility is handled